scripts/sample_qc.py

```diff
@@ -0,0 +1,332 @@
+import argparse
+import logging
+from typing import Optional, Union
+import hail as hl
+
+from gnomad.utils.filtering import filter_to_autosomes
+from gnomad.utils.annotations import bi_allelic_expr
+from gnomad.sample_qc.filtering import (
+    compute_stratified_sample_qc,
+    merge_sample_qc_expr,
+)
+from gnomad.sample_qc.sex import get_ploidy_cutoffs
+from gnomad.sample_qc.relatedness import (
+    compute_related_samples_to_drop,
+    DUPLICATE_OR_TWINS,
+    get_relationship_expr,
+    PARENT_CHILD,
+    SIBLINGS,
+    UNRELATED,
+)
+from gnomad.sample_qc.ancestry import pc_project
+
+from ccdg_qc.resources import *
+
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("CCDG_sample_qc")
+logger.setLevel(logging.INFO)
+
+
+def get_qc_vds(
+    data_type: str = "genomes",
+    pre_filter_variants: bool = False,
+    split: bool = False,
+    autosome_only: bool = False,
+    interval_qc: bool = False,
+) -> hl.vds.VariantDataset:
+    """
+    Wrapper function to get ccdg vds with desired filtering
+    :param data_type: Whether data is from genomes or exomes, default is genomes
+    :param split: Perform split on VDS, default is False
+    :param autosome_only: Whether to filter to variants in autosome, default is False
+    :param interval_qc: Whether to filter to high quality intervals for exomes QC, default is False
+    :return: ccdg vds with chosen filters
+    """
+    vds = hl.vds.read_vds(get_vds_path(data_type))
+    if (data_type == "exomes") & interval_qc:
```

Comment:
We're in Python rn - use `and` 😄 , and no need for parens

Reply:
😅